jib-core/src/test/java/com/google/cloud/tools/jib/docker/ImageToTarballTranslatorTest.java

```diff
@@ -60,10 +63,44 @@ public void testToTarballBlob()
         DescriptorDigest.fromHash(
             "5994471abb01112afcc18159f6cc74b4f511b99806da59b3caf5a9c173cacfc6");
 
-    Image<CachedLayer> testImage =
-        Image.<CachedLayer>builder()
-            .addLayer(new CachedLayer(fileA, new BlobDescriptor(fakeDigestA), fakeDigestA))
-            .addLayer(new CachedLayer(fileB, new BlobDescriptor(fakeDigestB), fakeDigestB))
+    Image<Layer> testImage =
+        Image.builder()
+            .addLayer(
+                new Layer() {
```

Comment:
If you prefer using a mock, you could do `.addLayer(mockLayer)` and `when(mockLayer.getBlob()).thenReturn(Blobs.from(fileA))`.

Reply:
I'll change to that to make the code less boiler-platey.